### PR TITLE
Update Site Affiliation article

### PR DESCRIPTION
**Pull request by agektmr**
Adds a form to request crawling.

### CODE REVIEW

**Review by BekzhanKassenov — APPROVED** [review]
LGTM

**Comment by agektmr** [comment]
@jpmedley @rachelandrew Can one of you review and merge this ASAP? If there's anything to change, feel free to apply it unless it's a drastic change. Thanks!

File: site/en/blog/site-affiliation/index.md [hunk 1]
```diff
@@ -167,9 +168,13 @@ at `/.well-known/assetlinks.json` on the respective domains.
     Allow: /.well-known/
     ```
 
+1.  Submit [a
```
Comment (samthor) [inline site/en/blog/site-affiliation/index.md:171]: I think you should end this with "/viewform" without the querystring so it doesn't automatically go into editing mode (if you have permissions to do that, which I guess is just Googlers).

**Review by samthor — APPROVED** [review]
LGTM from a code point of view

**Pull request merged**